sisl/nodes/context.py

```diff
@@ -1,36 +1,103 @@
 import contextlib
+from collections import ChainMap
+from typing import Any, Union
 
-def set_lazy_computation(nodes: bool = True, workflows: bool = True):
-    """Set the lazy computation mode for nodes and workflows.
+# The main sisl nodes context that all nodes will use by default as their base.
+MAIN_CONTEXT = dict(
```

Comment:
should we call this `SISL_NODE_CONTEXT` for clarity? Since this is importable the `MAIN` seems redundant?

Reply:
Done!